packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "15.2.0",
+  "version": "15.2.1",
```

Comment:
We probably won't be creating a release right after this is merged (we have a few other PRs coming that we can include in the next release) so do you mind reverting this? The web-components version bump is good though.